app/assets/javascripts/short-form/ShortFormNavigationService.js.coffee

```diff
@@ -40,24 +40,24 @@ ShortFormNavigationService = ($state, ShortFormApplicationService) ->
 
   Service.submitActions =
     'name': {path: 'contact'}
-    'contact': {callback: 'checkIfAddressVerificationNeeded'}
-    'verify-address': {path: 'alternate-contact-type'}
-    'alternate-contact-type': {callback: 'checkIfAlternateContactInfoNeeded'}
+    'contact': {callback: ['checkIfAddressVerificationNeeded', 'checkLiveWorkEligibility']}
```

Comment:
rather than writing a function to package these functions, I decided to just convert callback to an array and have the app iterate through multiple functions needed to run after submit.